configure.ac

```
@@ -54,7 +54,11 @@ AC_PROG_LN_S
 AC_PROG_MAKE_SET
 AM_DISABLE_STATIC
 AM_PROG_LIBTOOL
-AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+PKG_PROG_PKG_CONFIG
+dnl Give error and exit if we dont have pkgconfig
+if test "x$HAVE_PKGCONFIG" = "xno"; then
```

is this guaranteed to be set?  I don't see any mention of it in the manual page, and reading the macro's source code suggests there are at least some cases where it won't be set.  Actually, nothing references it there.

A better check seems to be whether `$PKG_CONFIG` is non-empty, which although not explicitly documented is the behavior of the macro.

```suggestion
if test "x$PKG_CONFIG" = "x"; then
```

---

cwendling is correct, you need to check if $PKG_CONFIG is set (to the full path of the build host's pkg-config executable. If it is unset, pkg-config is missing and builds should abort.